packages/router/src/config.ts

```diff
@@ -51,7 +51,7 @@ export const TChainConfig = Type.Object({
   providers: Type.Array(Type.String()),
   confirmations: Type.Number({ minimum: 1 }),
   defaultInitialGas: Type.Optional(TIntegerString),
-  subgraph: Type.String(),
+  subgraph: Type.Array(Type.String()),
```

Comment:
This will not be backwards compatible, you should either handle the type difference or add a new property called `subgraphs` thats an optional array (then convert the value in subgraph to a `string[]`)

Reply:
I was originally doing the latter, will reimplement

Reply:
So just to clarify however, I actually handle parsing the item passed into chainConfig.subgraph below, checking whether it's a string type, then coercing it to array per se if so. All of that is done before validation.

Why is it that that isn't backwards compat? Will it really error if I pass a string into subgraph, or won't it go through this coercing/formatting process first?

Reply:
Didn't realize the validation was done after the coercing, this will be fine